concordium-consensus/src/Concordium/Scheduler/EnvironmentImplementation.hs

```diff
@@ -136,6 +166,13 @@ instance (MonadReader ContextState m,
          )
          => SchedulerMonad pv (BSOMonadWrapper pv ContextState w state m) where
 
+  {-# INLINE insertTransactionVerificationResult #-}
+  insertTransactionVerificationResult txHash verResult = do
+    transactionVerificationCache %=! Map.insert txHash verResult
```

Comment:
I can't see where this function is used.

Reply:
It serves no purpose anymore. So I'll remove it.